include/zephyr/drivers/sensor.h

```diff
@@ -323,6 +323,8 @@ enum sensor_attribute {
 	 *  to the new sampling frequency.
 	 */
 	SENSOR_ATTR_FF_DUR,
+    /** Reference voltage in milli volts */
+    SENSOR_ATTR_REFERENCE_VOLTAGE_MV,
```

Comment:
Changing the API isn't ideal: it's quite hard to do in case we want this to go into upstream zephyr. 
It might be easier to use `SENSOR_ATTR_PRIV_START` instead

Reply:
I see your point. Should I create a header file like this: https://github.com/worldcoin/orb-mcu-firmware/blob/04c427809c032bc831cf3a554a0ee9d7eeb78a08/drivers/sensor/stm32_temp_tfh/stm32_temp_tfh.h
... or just use "SENSOR_ATTR_PRIV_START", which is not very intuitive to use.

Reply:
yeah add a definition in the sensor header file as done in the other sensors that make use of `SENSOR_ATTR_PRIV_START`

Reply:
Done. @fouge can you please double check?